atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -374,6 +330,34 @@ def update_repos_from_composes(self):
             self.log.info('adding repo file from compose: %s', compose_info['result_repofile'])
             self.repos.append(compose_info['result_repofile'])
 
+    def _add_filesystem_to_dockerfile(self, file_path, build_dir: BuildDir):
+        """
+        Put an ADD instruction into the Dockerfile (to include the filesystem
+        into the container image to be built)
+        """
+        content = 'ADD {0} /'.format(file_path)
```

Comment:
The path needs to be relative:
```dockerfile
ADD filesystem.tar.gz /
```
The current implementation would use this path, which wouldn't work
```dockerfile
ADD /workspace/build-dir/filesystem.tar.gz /
```

Reply:
updated.